src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8" />
+	<title>JAvaScript</title>
+	<meta name="keywords" content="HTML, CSS, JavaScript" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+<body>
+<h1>Hello world!</h1>
+
+<p style="cursor: pointer; padding: 5px; font-size: 24px; background-color: #000000" id="text1">Text 1</p>
+<p style="cursor: pointer; padding: 5px; font-size: 24px; background-color: #000000" id="text2">Text 2</p>
+<p style="cursor: pointer; padding: 5px; font-size: 24px; background-color: #000000" id="text3">Text 3</p>
+<script type="module" src="scripts/main.js"></script>
```

Comment:
Do you need module type here? Unlike classic scripts, module scripts require the use of the CORS protocol for cross-origin fetching.
"text/javascript" should work well.

Reply:
I wrote it as in the video tutorial when it was necessary to connect export / import

Reply:
Do you use export/import in your project?